modules/features/falcon_feature_user/config/install/user.role.donations_manager.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies: {  }
+id: donations_manager
+label: 'Donations manager'
+weight: 5
+is_admin: null
+permissions:
+  - 'access toolbar'
```

Comment:
Correct me if I'm wrong but I'd give this user the permissions to view orders (at least view).

Reply:
Granted permissions.